pallets/profiles/src/rpc.rs

```diff
@@ -78,4 +78,10 @@ impl<T: Config> Module<T> {
     pub fn get_account_data(account: T::AccountId) -> T::AccountData {
         SystemPallet::<T>::account(&account).data
     }
+
+    pub fn filter_followed_accounts(account: T::AccountId, maybe_following: Vec<T::AccountId>) -> Vec<T::AccountId> {
```

Comment:
why does the name has a `maybe_` prefix? the `Vec` type already carries the possibility of it being empty. so no need to explicitly state in the name that it can be empty.

Reply:
`maybe_` prefix here is because not every account in the array may follow target `account`.

Lines 83-85:
```rust
maybe_following.iter()
            .filter(|maybe_following| Self::account_followed_by_account((&account, maybe_following)))
            .cloned().collect()
```

The code above says: does `&account` follows `maybe_following`?